src/components/warning-component.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import colors from '@styles/colors';
+
+enum warningType {
+    Thunderstorm,
+    Strong_wind,
+    Snowstorm,
+    Rain,
+    Fog,
+    Little_wind,
+    hail,
+}
+
+export function Warning(War: warningType) {
```

Comment:
We use the result of function only in this component so you don't need to export it now. As functions we usually use arrow functions, for example: 
`const getSomething = (some: SomeType) => {
    /* do something */
    return value;
}`
Also rename function name, variable to lower case and i think it's better to name function as getWarningText
![image](https://user-images.githubusercontent.com/52620425/223988001-bb606b12-4fe6-4ea8-8dd3-b274aa120732.png)